bundles/org.openhab.binding.qbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="qbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Qbus Bridge</label>
+		<description>This bridge represents a Qbus client</description>
+		<config-description>
+			<parameter name="addr" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>IP Address of Qbus server, usually 'localhost'</description>
+				<default>localhost</default>
+				<advanced>false</advanced>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="sn" type="text" required="true">
+				<label>Serial nr</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Can you work on this? Please check all.